CHANGELOG.md

```diff
@@ -0,0 +1,124 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+## March 19, 2018    
+
+### Removed
+
+* Change log from README.md.  
+
+### Added   
+
+* CHANGELOG.md tracking all the changes to the project.   
+* Link to CHANGELOG.md in README.md.   
+
+
+## March 6, 2018    
+
+### Changed  
+
+* README.md - Added change log to README.md for iotsp-device-sdk-common-0.3.3.0.jar.
+
+### Added  
+
+* iotsp-device-sdk-common-0.3.3.0.jar - API to remove the supported commands from the GwConfigCache. Once the command is removed, GwConfigCache will throw an exception when command of that type is used to update cache. Users could add the command back using addSupportedCommandToCache. Corresponding javadoc in iotsp-device-sdk-common-0.3.3.0-SNAPSHOT-javadoc.jar.       
```

Comment:
API to remove the supported commands from the GwConfigCache. Once the command is removed, GwConfigCache will throw an exception when command of that type is used to update cache. Users could add the command back using addSupportedCommandToCache.

Corresponding javadoc in iotsp-device-sdk-common-0.3.3.0-SNAPSHOT-javadoc.jar.